src/h5web/dataset-visualizer/DatasetVisualizer.tsx

```diff
@@ -17,11 +18,14 @@ function DatasetVisualizer(props: Props): JSX.Element {
 
   return (
     <div className={styles.visualizer}>
-      <VisSelector
-        activeVis={activeVis}
-        choices={supportedVis}
-        onChange={setActiveVis}
-      />
+      <div className={styles.toolbar}>
```

Comment:
Just thinking that the term you used for the name of the branch may be better suited: `visBar`. This way we can distinguish the three elements: the parent `visBar`, the `VisSelector` and the `VisToolbar`.